Basic-Car-Maintenance/Shared/Dashboard/Views/DashboardView.swift

```diff
@@ -123,12 +130,21 @@ struct DashboardView: View {
                     Menu {
                         Picker(selection: $viewModel.sortOption) {
                             ForEach(DashboardViewModel.SortOption.allCases) { option in
-                                Text(option.label)
-                                    .tag(option)
+                                if option != .vehicleType {
+                                    Text(option.label)
+                                        .tag(option)
+                                }
                             }
                         } label: {
                             EmptyView()
                         }
+                        
+                        Button(action: {
+                            viewModel.sortOption = .vehicleType
+                            isShowingVehicleSelection = true
+                        }, label: {
+                            Text(DashboardViewModel.SortOption.vehicleType.label)
+                        })
```

Comment:
personally I like multiple trailing closure syntax, I make this comment often, do you think I should write this down somewhere?

like this:
```swift
Button {
    // action here
} label: {
}
```